user/views.py

```diff
@@ -10,16 +8,33 @@
 class CreateUserView(generics.CreateAPIView):
     serializer_class = UserSerializer
 
+    def post(self, request, *args, **kwargs):
+        """Created a new user"""
+        return self.create(request, *args, **kwargs)
+
 
 class CreateTokenView(ObtainAuthToken):
     renderer_classes = api_settings.DEFAULT_RENDERER_CLASSES
-    serializer_class = AuthTokenSerializer
+
+    def post(self, request, *args, **kwargs):
+        """Login endpoint"""
+        return super().post(request, *args, **kwargs)
 
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAuthenticated,)
 
     def get_object(self):
         return self.request.user
+
+    def get(self, request, *args, **kwargs):
+        """Retrieve information about current user"""
+        return super().get(request, *args, **kwargs)
+
+    def put(self, request, *args, **kwargs):
+        """Update all information about current users"""
+        return super().get(request, *args, **kwargs)
```

Comment:
The `put` method in `ManageUserView` should call `super().update(request, *args, **kwargs)` instead of `super().get(request, *args, **kwargs)` to correctly handle full updates.